### PR TITLE
Fix submitResponse with empty scope

### PR DESCRIPTION
**Pull request by daveroga**
- Fix submitResponse with empty scope
- Add integration test with empty scope

### CODE REVIEW

File: src/storage/interfaces/onchain-zkp-verifier.ts [hunk 1]
```diff
@@ -51,14 +51,14 @@ export interface IOnChainZKPVerifier {
    * @param {AuthProofResponse} authResponse - authResponse
    * @param {ZeroKnowledgeProofMultiQueryResponse[]} responses - singleResponses and groupedResponses
    * @param {ZeroKnowledgeProofAuthResponse} [authProof] - authProof in case of authV2
-   * @returns {Promise<Map<string, ZeroKnowledgeInvokeResponse>>} - map of transaction hash - ZeroKnowledgeInvokeResponse
+   * @returns {Promise<{ txHash: string; responsesMap: Map<string, ZeroKnowledgeInvokeResponse> }>} - transaction hash and map of transaction hash - ZeroKnowledgeInvokeResponse
```
Comment (volodymyr-basiuk) [inline src/storage/interfaces/onchain-zkp-verifier.ts:54]: looks like breaking changes, but we marked as `@beta`..

**Review by volodymyr-basiuk — APPROVED** [review]

**Review by vmidyllic — COMMENTED** [review]
Lgtm

File: src/storage/interfaces/onchain-zkp-verifier.ts (hunk 1 above)
Comment (daveroga) [reply src/storage/interfaces/onchain-zkp-verifier.ts]: Solved
